Release notes for monitoring agent RPMs

agrawalkaushik opened the pull request:
Release notes for monitoring agent 1.1 and 2.0 releases.

MichaelBaj commented on sidebars.js:
```diff
@@ -119,6 +119,14 @@ module.exports = {
+        "label": "128T Monitoring Agent",
+        "items": [
+          "release_notes_monitoring_agent_1.1",
+          "release_notes_monitoring_agent_2.0",
```
these should be in reverse order.  The newest should always be on top.

MichaelBaj commented on docs/release_notes_monitoring_agent_1.1.md:
```diff
@@ -0,0 +1,39 @@
+title: 128T Monitoring Agent 1.1 Release Notes
+sidebar_label: 1.1
+---
+# Release 1.1.0
```
headings should start with ## - as defined by the CONTRIBUTING.md#guidelines

MichaelBaj commented on docs/release_notes_monitoring_agent_1.1.md:
```diff
@@ -0,0 +1,39 @@
+---
+# Release 1.1.0
+
+# New Features and Improvements
```
should be ### h3

MichaelBaj commented on docs/release_notes_monitoring_agent_1.1.md:
```diff
@@ -0,0 +1,39 @@
+
+# New Features and Improvements
+
+### MON-148 Top applications, sessions and sources input plugin
```
Following the pattern in other release notes these should be bulleted items just like ## Issues Fixed

agrawalkaushik replied:
I wanted to try this as the features get better highlighted this way. I felt it improves the readability a bit

MichaelBaj commented on docs/release_notes_monitoring_agent_1.1.md:
```diff
@@ -0,0 +1,39 @@
+The `t128_lte_metric` collector will look for and report SNR signal strength if its reported by the 128T router.
+
+
+# Issues Fixed
```
This needs to be ## h2

MichaelBaj commented on docs/release_notes_monitoring_agent_1.1.md:
```diff
@@ -0,0 +1,39 @@
+# New Features and Improvements
+
+### MON-148 Top applications, sessions and sources input plugin
+To conflgure the new input plugin, please read the [Monitoring Agent Guide](plugin_monitoring_agent.md)
```
typo:  configure

ptimmons commented on docs/release_notes_monitoring_agent_1.1.md:
```diff
@@ -0,0 +1,39 @@
+# New Features and Improvements
+
+### MON-148 Top applications, sessions and sources input plugin
+To conflgure the new input plugin, please read the [Monitoring Agent Guide](plugin_monitoring_agent.md)
```
configure

agrawalkaushik replied:
Fixed

MichaelBaj commented on docs/release_notes_monitoring_agent_2.0.md:
```diff
@@ -0,0 +1,42 @@
+title: 128T Monitoring Agent 2.0 Release Notes
+sidebar_label: '2.0'
+---
+# Release 2.0.0
```
must be ## h2

MichaelBaj commented on docs/release_notes_monitoring_agent_2.0.md:
```diff
@@ -0,0 +1,42 @@
+---
+# Release 2.0.0
+
+# New Features and Improvements
```
Must be ## h2

MichaelBaj commented on docs/release_notes_monitoring_agent_2.0.md:
```diff
@@ -0,0 +1,42 @@
+### MON-171 Update Telegraf to latest stable version 1.14.0 ####
+A new stable version of telegraf was released upstream with several new inputs such as execd, wireguard and others.
+
+### MON-175 LTE metric collect will include SNR signal strength ####
```
Following the pattern in other release notes these should be bulleted items just like ## Issues Fixed

ptimmons commented on docs/release_notes_monitoring_agent_1.1.md:
```diff
@@ -0,0 +1,39 @@
+A new stable version of telegraf was released upstream with several new inputs such as execd, wireguard and others.
+
+### MON-175 LTE metric collect will include SNR signal strength ####
+The `t128_lte_metric` collector will look for and report SNR signal strength if its reported by the 128T router.
```
"it is", not "its"

MichaelBaj commented on docs/release_notes_monitoring_agent_2.0.md:
```diff
@@ -0,0 +1,42 @@
+----
+- **MON-125** `t128_metrics` default bfd config doesn't work with 4.3
+
+  _**Resolution:**_ The new default config for metrics have the correct parameters for bfg metrics
```
typo: bfd metrics

ptimmons replied:
BFD, not "bfg"

MichaelBaj commented on docs/release_notes_monitoring_agent_2.0.md:
```diff
@@ -0,0 +1,42 @@
+
+  _**Resolution:**_ The new default config for metrics have the correct parameters for bfg metrics
+----
+- **MON-146** Metric collector timing out with the default config on customer system
```
remove "on customer system"

MichaelBaj commented on docs/release_notes_monitoring_agent_1.1.md:
```diff
@@ -0,0 +1,39 @@
+# Issues Fixed
+
+------
+- **MON-146** Metric collector timing out with the default config on customer system
```
remove "on customer system"

ptimmons commented on docs/release_notes_monitoring_agent_1.1.md:
```diff
@@ -0,0 +1,39 @@
+------
+- **MON-160** sample agent-config has invalid tags
+
+  _**Resolution:**_ All the sample configuration now contain valid data
```
"configurations"

ptimmons commented on docs/release_notes_monitoring_agent_1.1.md:
```diff
@@ -0,0 +1,39 @@
+
+  _**Resolution:**_ All peer paths on the node will be reported by the peer-path collector
+------
+- **MON-170** Default telegraf service (not 128T-telegraf) is enabled and running un-necessarily on the system
```
unnecessarily

ptimmons commented on docs/release_notes_monitoring_agent_2.0.md:
```diff
@@ -0,0 +1,42 @@
+# New Features and Improvements
+
+### MON-148 Top applications, sessions and sources input plugin
+To conflgure the new input plugin, please read the [Monitoring Agent Guide](plugin_monitoring_agent.md)
```
configure

ptimmons commented on docs/release_notes_monitoring_agent_2.0.md:
```diff
@@ -0,0 +1,42 @@
+------
+- **MON-160** sample agent-config has invalid tags
+
+  _**Resolution:**_ All the sample configuration now contain valid data
```
configurations

ptimmons commented on docs/release_notes_monitoring_agent_2.0.md:
```diff
@@ -0,0 +1,42 @@
+
+  _**Resolution:**_ All peer paths on the node will be reported by the peer-path collector
+------
+- **MON-170** Default telegraf service (not 128T-telegraf) is enabled and running un-necessarily on the system
```
unnecessarily

MichaelBaj commented:
Also, I added integration between Jira and GitHub so you should be able to use smart commit hooks in your comments to associate this with the corresponding Jira issue ID.

agrawalkaushik commented:
> Also, I added integration between Jira and GitHub so you should be able to use smart commit hooks in your comments to associate this with the corresponding Jira issue ID.

That is cool - I just created a task and added the JIRA id to my latest commit.

MichaelBaj commented:
Kaushik addressed PT's changed in his latest commit.
